libfastpm/cosmology.c

```diff
@@ -323,7 +333,21 @@ static ode_soln growth_ode_solve(double a, FastPMCosmology * c)
     return soln;
 }
 
-/* FIXME: Each of these functions will redo the ode! That's long! */
+void fastpm_growth_info_init(FastPMGrowthInfo * growth_info, double a, FastPMCosmology * c) {
+    ode_soln soln = growth_ode_solve(a, c);
+    ode_soln soln_a1 = growth_ode_solve(1, c);
+    // FIXME: you could save a=1 soln at the start of code (perhaps to cosmo obj) and then never compute again.
```

Comment:
Proabley better this way, hiding the absolute z=0 value to make sure we don't use it by mistake.